cmd/bridge/config.go

```diff
@@ -41,6 +41,9 @@ type ClusterInfo struct {
 	ConsoleBaseAddress string `yaml:"consoleBaseAddress"`
 	ConsoleBasePath    string `yaml:"consoleBasePath"`
 	MasterPublicURL    string `yaml:"masterPublicURL"`
+	// DeveloperPublicConsoleURL is an optional URL of the developer console. If specified, a context switcher is
+	// added to the masthead with a link to the developer console. This option will be removed in a future release.
+	DeveloperPublicConsoleURL string `yaml:"developerConsolePublicURL"`
```

Comment:
new field lgtm

Reply:
> new field lgtm

remember that you'll need to leave it inert for one release to allow a clean update and rollback.